README.md

```diff
@@ -1,17 +1,23 @@
 # group-11
 
-Imagine you rob a bank, the plan goes perfectly and nobody manages to stop you while you exit the building with all the money you just proudly stole. BUT you missed one small detail of the robbery plan... THE GETAWAY! You did not think about how you're going to escape town with your car at all and the police is everywhere. Dont even get me started on the civilians being all over the place! You must now use your driving skills and prove worthy of all the money you just got and the title of the best robber of all time.
 
+# What 
+We are creating an application to drive the car maunally for seeking treasure(s). The car will be able to take multiple routes to find the treasure. The environment will consist of multiple objects/obstacles and one treasure. The user will be alerted when there is an object nearby and the car will be stopped if it gets too close to the object. The closer or further the car is from the treasure will be displayed on the app. The user will be able to view the car's speed and how much the car has travelled so far. The user will have a first person view (driver seat view). 
```

Comment:
What will be the treasure?
How will the treasures be "detected"? (with or without quotes)

Reply:
- The treasure will be an object different from all the other objects.
- The treasure will be detected by moving in place but the other objects will be still, therefore we will be using an infrared sensor.